wiki/Tournaments/oPHN/2022_catch/en.md

```diff
@@ -0,0 +1,331 @@
+| **Henrick** ::{ flag=CA }:: | **4** | 0 | ::{ flag=PH }:: Puro and Colin | [#1](https://osu.ppy.sh/community/matches/99835436) |
+| **- Kielzu -** ::{ flag=PH }:: | **4** | 0 | ::{ flag=PH }:: arwab | [#1](https://osu.ppy.sh/community/matches/99804602) |
+| **Mawile** ::{ flag=PH }:: | **4** | 2 | ::{ flag=PH }:: Laqure | [#1](https://osu.ppy.sh/community/matches/99799498) |
+| Kyonkichi ::{ flag=PH }:: | 3 | **4** | ::{ flag=PH }:: **Kuroiden** | [#1](https://osu.ppy.sh/community/matches/99798469) |
+
+### Round of 16
+
+| Player 1 |  |  | Player 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **Laqure** ::{ flag=PH }:: | **4** | 0 | ::{ flag=PH }:: Kuroiden | [#1](https://osu.ppy.sh/community/matches/99603839) |
+| **Puro and Colin** ::{ flag=PH }:: | **4** | 0 | ::{ flag=PH }:: slvn_pedro | [#1](https://osu.ppy.sh/community/matches/99600977) |
+| **-Rustyy** ::{ flag=PH }:: | **4** | 0 | ::{ flag=PH }:: CoolEVT | [#1](https://osu.ppy.sh/community/matches/99569808) |
+| -HyperDream ::{ flag=PH }:: | 0 | **4** | ::{ flag=PH }:: **arwab** | [#1](https://osu.ppy.sh/community/matches/99563126) |
+| **Mawile** ::{ flag=PH }:: | **4** | 0 | ::{ flag=PH }:: Kyonkichi | [#1](https://osu.ppy.sh/community/matches/99561985) |
+| **Jinyori** ::{ flag=PH }:: | **4** | 3 | ::{ flag=PH }:: Chloebe | [#1](https://osu.ppy.sh/community/matches/99559982) |
+
+## Ruleset
+
+### General rules
+
+- ScoreV2 will be used for this tournament.
+- Players are allowed to join the 10 minutes before the scheduled match.
+- Players cannot nullify a beatmap under any circumstances
+- Players will be given 10 minutes to join the lobby, failing to show up will result to win by default(WBD).
+- The first 30 seconds of the match can be aborted by the referee if any unfortunate event happen in player's end such as sudden internet disconnections or blackout this is only allowed once in a match per player/participants and they will be given 3 minutes to get back in order to restart the round.
+- Players are allowed to join the 10 minutes before the scheduled match.
+- Players cannot nullify a beatmap under any circumstances
+- Players will be given 10 minutes to join the lobby, failing to show up will result to win by default(WBD).
+
+### Qualifier guidelines
+
+- In order to advance to the knockout stage, Players need to accumulate enough scores for each mod bracket in order to be able to proceed to knockout stage.
+- Participants will be split into lobbies with different time within the day.
+- Participants has the discretion to change their lobbies 2 days before the start of the qualifiers.
+- As the qualifiers on each lobby starts, the map pool will traverse in this pattern:
+  - NM 1 > HD 1 > HR 1 > DT 1 > NM 2 > HD 2 > HR 2 > DT 2
+- 2 Rotations will occur per each session.
+- If a tied overall qualifier scores occurred, the seeding of each involved player will based on their current performance points as of the day when the qualifiers started
```

Comment:
```suggestion
- In order to advance to the knockout stage, players need to accumulate enough scores for each mod bracket.
- Participants will be split into lobbies for different times during the day.
- Participants have the discretion to change their lobbies 2 days before the start of the Qualifiers.
- As the Qualifiers on each lobby starts, the map pool will follow this pattern:
  - NM 1 -> HD 1 -> HR 1 -> DT 1 -> NM 2 -> HD 2 -> HR 2 -> DT 2
- 2 rotations will occur per each session.
- In case of ties in Qualifier scores, the seeding of affected players will be based on their current performance points as of the day when the Qualifiers started.
```